examples/http-timeout/http_timeout.metatags

```diff
@@ -1,2 +1,2 @@
-description: BBE on how to use an HTTP timeout in Ballerina. This will set a timeout for the requests to get a response and will return an error if a response is not returned within the given timeout.
+description: This Ballerina example is on how to use an HTTP timeout in Ballerina. This will set a timeout for the requests to get a response and will return an error if a response is not returned within the given timeout.
```

Comment:
```suggestion
description: This example is on how to use an HTTP timeout in Ballerina. This will set a timeout for the requests to get a response and will return an error if a response is not returned within the given timeout.
```